narwhals/dataframe.py

```diff
@@ -857,28 +877,57 @@ def drop(self, *columns: str | Iterable[str]) -> Self:
             *columns: Names of the columns that should be removed from the dataframe.
 
         Examples:
-            >>> import pandas as pd
+            Drop a single column by passing the name of that column.
+
```

Comment:
this docstring was fine already - I don't think we need to modify it

Reply:
Concerning this, I haven't made any changes to the docstrings to this method. From my end, they are as they have been since the last mod from George and Franco. Please is there any particular change you may have noticed?

Reply:
you made the changes which I commented on here 😄